packages/styles/scss/components/leadspace/_leadspace.scss

```diff
@@ -488,19 +430,40 @@ $btn-min-width: 26;
     }
   }
 
+  :host(#{$dds-prefix}-leadspace) ::slotted([slot='extra-elements']) {
+    z-index: 1;
+    padding-left: 1rem;
+    padding-bottom: 0.5rem;
+    max-width: 640px;
+  }
+
   :host(#{$dds-prefix}-leadspace-heading),
   .#{$prefix}--leadspace__title {
     padding-top: 0;
     margin-bottom: $spacing-09;
     padding-left: 1rem;
```

Comment:
Use spacing token here.

Reply:
update all the tokens and added rem! thanks Kenny!